_events/2023-01-25-htcondor-osg.md

```diff
@@ -1,11 +1,11 @@
 ---
-short_title: Joint OSG All Hands Meeting and HTCondor Week July 10-14
-title: "Save the Dates for the Joint HTCondor Week - OSG All-Hands Meeting - 10-14 July 2023"
+short_title: Announcing Throughput Computing 2023 to be held July 10-14
+title: "Save the Dates for Throughput Computing 2023 - a joint HTCondor OSG All-Hands event"
 layout: events
 published: true
 
 excerpt: |
-    Joint OSG All Hands Meeting and HTCondor Week July 10-14 2023
+    Save the Dates for **Throughput Computing 2023** - a joint OSG All Hands Meeting and HTCondor Week to be held July 10-14 2023 in Madison, Wisconsin
```

Comment:
Janet, it should always be “OSG All-Hands Meeting” — exactly like that, with the hyphen. This is our brand name, so just like any brand name, we want it to appear the same way every time.